### ISSUE TITLE
Reported a11y issue: Some code examples use a colour that has very low contrast with the background colour

### ISSUE THREAD

**Issue by richardTowers** [alphagov/tech-docs-gem#213]
This was reported in [a recent accessibility audit](https://docs.google.com/spreadsheets/d/1s53eGMBj2hdTVLRitwi673R-_uERWh0kOxOp9QsOVg0/edit#gid=0)

I'm not 100% it's true though. Looking at all the highlight colours in the stylesheet with:

```html
<pre class="highlight ruby"><code><span class="cs">cs</span>	<span class="kr">kr</span>	<span class="kn">kn</span>	<span class="kt">kt</span>
<span class="ow">ow</span>	<span class="py">py</span>	<span class="nn">nn</span>	<span class="nf">nf</span>	<span class="vi">vi</span>
<span class="nd">nd</span>	<span class="nx">nx</span>	<span class="ne">ne</span>	<span class="l">l</span>	<span class="ld">ld</span>
<span class="mo">mo</span>	<span class="sh">sh</span>	<span class="ss">ss</span>	<span class="si">si</span>	<span class="sd">sd</span>
<span class="sc">sc</span>	<span class="gi">gi</span>	<span class="gd">gd</span>	<span class="p">p</span>	<span class="w">w</span>
<span class="hll">hll</span>	<span class="gt">gt</span>	<span class="gs">gs</span>	<span class="ge">ge</span>	<span class="gh">gh</span>
<span class="gu">gu</span>	<span class="gp">gp</span></code></pre>
```

Renders as:

![image](https://user-images.githubusercontent.com/1696784/109996724-68b70500-7d07-11eb-8056-4d7269eace14.png)

Firefox's Accessibility tools report that all of these are at least AA compliant, so I'm not sure this is a real issue.

I'm raising it here as a better place for discussion than a spreadsheet. If we don't think it's reproducible then we can probably close it and cross it off our list of issues.

**Comment by jonathanglassman** [comment]
@selfthinker can you advise?

**Comment by 36degrees** [comment]
When was the audit done / what version of the gem was it against?

Is there any chance it was resolved by https://github.com/alphagov/tech-docs-gem/pull/189? Looks like it shipped in 2.0.13.

**Comment by richardTowers** [comment]
Possibly. The audit was done in August, which is just a little bit after that shipped. I can't find a reference to this in [the original report](https://drive.google.com/file/d/1aXLwlZmLTY9joHGLTJ_X7L_sMwPXjhFg/view), so I'm inclined to think it's not a real issue.
